src/PlatformEngines.jl

```diff
@@ -279,8 +297,13 @@ function download(
     else
         (total, now) -> nothing
     end
-    try
-        Downloads.download(url, dest; headers, progress)
+    return try
+        if verbose && !do_fancy
+            response = Downloads.request(url; output = dest, headers, verbose)
+        else
+            response = Downloads.request(url; output = dest, headers, progress)
```

Comment:
It seems strange to me that we might not pass `verbose` through here, despite the fact that it could be `true`.

Reply:
Yeah. The problem is that I'm trying to preserve the current behavior of `PlatformEngines.download`. Currently, if you pass `verbose` to `PlatformEngines.download`, it only gets used to determine whether or not we show a progress bar; it never gets passed through to `Downloads.download.` The `Downloads.download` call is `Downloads.download(url, dest; headers, progress)`.